core-primitives/sgx-runtime-primitives/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "itp-sgx-runtime-primitives"
+version = "0.9.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2021"
+
+[dependencies]
+
+# Substrate dependencies
+frame-system = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
+pallet-balances = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
+sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
+
+[features]
+default = ["std"]
+std = [
+    "frame-system/std",
```

Comment:
Can you add the pallet-balances here?

Reply:
Thank you, nice catch, I only added the frame-system when it was added, that's why it was not there (yet).
Fixed in f347d9ec7f4da741eca5e4a36bac988be8d94aab.